lib/assert.js

```diff
@@ -110,15 +111,23 @@ function fail(actual, expected, message, operator, stackStartFn) {
       );
     }
     if (argsLen === 2)
-      operator = '!=';
+      operator = defaultOp;
```

Comment:
This if seems obsolete now. With two arguments `operator` and `message` will always be undefined and the default will be chosen below.